pages/windows/sc.md

```diff
@@ -0,0 +1,20 @@
+# sc
+
+> Communicate with the Service Control Manager and services on Windows from the command-line.
```

Comment:
```suggestion
> Communicate with the Service Control Manager and services.
```

Reply:
Regarding this, we might need to fix it in other Windows pages too.